src/components/customers/ProfileFollowUp.vue

```diff
@@ -227,7 +227,7 @@ export default {
     await this.refreshCustomer();
     const promises = [this.getUserHelpers(), this.getAuxiliaries()];
     if (this.customer.firstIntervention) promises.push(this.getCustomerFollowUp());
-    if (this.customer.fundings.length) promises.push(this.getCustomerFundingsMonitoring(this.customer._id));
+    if (this.customer.fundings.length) promises.push(this.getCustomerFundingsMonitoring());
```

Comment:
il faut que tu passes la `query` non ? `{ customer: this.customer._id }`

Reply:
Non car ici ca appelle la méthode de la page

Reply:
ha oui pardon ><